neptune/new/internal/init/model.py

```diff
@@ -55,7 +59,17 @@ def init_model(
     flush_period: float = DEFAULT_FLUSH_PERIOD,
     proxies: Optional[dict] = None,
 ) -> Model:
-    verify_type("model", model, (str, type(None)))
+    if model:
+        if with_id:
+            raise NeptuneInitParametersCollision("with_id", "model", method_name="init_model")
+
+        _logger.warning(
+            "parameter `model` is deprecated, use `with_id` instead."
+            " We'll end support of it in `neptune-client==1.0.0`."
```

Comment:
Do we need to do it like this? Is't there some cool standard decorator?

Reply:
There are some tools to mark some "things" as deprecated, but we don't use them. There's still issue for that in Jira.
Here I've used just ad-hoc solution, choosing some library will require substantial research on technlogy which'll suit best our deprecation use-cases (mostly imports).

Reply:
> but we don't use them

Maybe it's good moment to start. Now there is a lot of awful code duplication. If you prefer, you can create some very simple decorator by yourself. But I belive there is some simple lib for such things.

Reply:
Ok, let's create simple decorator for now.